types.go

```diff
@@ -156,11 +156,26 @@ type ProjectUpdate struct {
 
 // Project represents the awx api project.
 type Project struct {
```

Comment:
The project struct is also not same to the awx rest api, could you help me clear it?

```
{
    "id": 4,
    "type": "project",
    "url": "/api/v2/projects/4/",
    "related": {
        "named_url": "/api/v2/projects/Demo Project++Default/",
        "created_by": "/api/v2/users/1/",
        "last_job": "/api/v2/project_updates/3757/",
        "notification_templates_error": "/api/v2/projects/4/notification_templates_error/",
        "notification_templates_success": "/api/v2/projects/4/notification_templates_success/",
        "object_roles": "/api/v2/projects/4/object_roles/",
        "notification_templates_any": "/api/v2/projects/4/notification_templates_any/",
        "copy": "/api/v2/projects/4/copy/",
        "project_updates": "/api/v2/projects/4/project_updates/",
        "update": "/api/v2/projects/4/update/",
        "access_list": "/api/v2/projects/4/access_list/",
        "teams": "/api/v2/projects/4/teams/",
        "scm_inventory_sources": "/api/v2/projects/4/scm_inventory_sources/",
        "inventory_files": "/api/v2/projects/4/inventories/",
        "schedules": "/api/v2/projects/4/schedules/",
        "playbooks": "/api/v2/projects/4/playbooks/",
        "activity_stream": "/api/v2/projects/4/activity_stream/",
        "organization": "/api/v2/organizations/1/",
        "last_update": "/api/v2/project_updates/3757/"
    },
    "summary_fields": {
        "last_job": {
            "id": 3757,
            "name": "Demo Project",
            "description": "",
            "finished": "2018-06-15T09:33:35.856828Z",
            "status": "successful",
            "failed": false
        },
        "last_update": {
            "id": 3757,
            "name": "Demo Project",
            "description": "",
            "status": "successful",
            "failed": false
        },
        "organization": {
            "id": 1,
            "name": "Default",
            "description": ""
        },
        "created_by": {
            "id": 1,
            "username": "admin",
            "first_name": "Kai",
            "last_name": "Ying"
        },
        "object_roles": {
            "admin_role": {
                "id": 13,
                "description": "Can manage all aspects of the project",
                "name": "Admin"
            },
            "use_role": {
                "id": 15,
                "description": "Can use the project in a job template",
                "name": "Use"
            },
            "update_role": {
                "id": 16,
                "description": "May update project or inventory or group using the configured source update system",
                "name": "Update"
            },
            "read_role": {
                "id": 14,
                "description": "May view settings for the project",
                "name": "Read"
            }
        },
        "user_capabilities": {
            "edit": true,
            "start": true,
            "copy": true,
            "schedule": true,
            "delete": true
        }
    },
    "created": "2018-05-22T06:08:07.848477Z",
    "modified": "2018-06-15T09:33:35.868448Z",
    "name": "Demo Project",
    "description": "",
    "local_path": "_4__demo_project",
    "scm_type": "git",
    "scm_url": "https://github.com/ansible/ansible-tower-samples",
    "scm_branch": "",
    "scm_clean": false,
    "scm_delete_on_update": false,
    "credential": null,
    "timeout": 0,
    "last_job_run": "2018-06-15T09:33:35.856828Z",
    "last_job_failed": false,
    "next_job_run": null,
    "status": "successful",
    "organization": 1,
    "scm_delete_on_next_update": false,
    "scm_update_on_launch": true,
    "scm_update_cache_timeout": 0,
    "scm_revision": "347e44fea036c94d5f60e544de006453ee5c71ad",
    "custom_virtualenv": null,
    "last_update_failed": false,
    "last_updated": "2018-06-15T09:33:35.856828Z"
}
```

Reply:
Sure. Some fields are missing in the structure. Will fix it now and test in my local repository.

Reply:
Okay, thanks!